src/entity/User.ts

```diff
@@ -150,9 +150,16 @@ export const updateUserEmail = async (
 
   return con.transaction(async (entityManager) => {
     try {
-      await entityManager
+      const res = await entityManager
         .getRepository(User)
         .update({ id: data.id }, { email: data.email });
+      if (res.affected === 0) {
```

Comment:
Should we care about this edge case even?

Reply:
I guess it can't harm to know if it ever happens, most likely won't ever be hit, but still